packages/http-client-python/emitter/src/lib.ts

```diff
@@ -83,6 +83,48 @@ const libDef = {
         default: "Can't generate Python SDK since no client defined in typespec file.",
       },
     },
+    "no-valid-paging-items": {
+      severity: "warning",
+      messages: {
+        default: paramMessage`No valid paging items for operation '${"operationId"}'.`,
+      },
+    },
+    "no-valid-nextlink": {
+      severity: "warning",
+      messages: {
+        default: paramMessage`No valid next link for operation '${"operationId"}'.`,
+      },
+    },
+    "no-valid-lro-result": {
+      severity: "warning",
+      messages: {
+        default: paramMessage`No valid LRO result for operation '${"operationId"}'.`,
+      },
+    },
+    "no-valid-wire-name-for-continuation-token-in-request": {
```

Comment:
we could combine these two and just call them `invalid-continuation-token` and in the message say if it's request or response